app/utils/PostcodeHelper.scala

```diff
@@ -0,0 +1,9 @@
+package utils
+
+import uk.gov.hmrc.address.uk.Postcode
+
+object PostcodeHelper {
+
+  def displayPostcode(p:Option[String]) = p.fold("")(pc => Postcode.cleanupPostcode(pc).fold("")(_.toString))
+  def displayPostcode(p:String) = Postcode.cleanupPostcode(p).fold("")(_.toString)
```

Comment:
:D this is become a bit of an early-morning code-golf. 

```scala
   def displayPostcode(p:String) = Postcode.cleanupPostcode(p).map(_.toString).getOrElse("")
```

Reply:
done